### PR TITLE
feat: create UI components dropdown-menu

### PR DESCRIPTION
**Pull request by yuki-92c**
## Overview

- Add dropdown menu component in the components/ui folder based on the design on Figma.

## Changes

- Install dropdown menu component from shadcn/ui
    - https://ui.shadcn.com/docs/components/dropdown-menu
- Style the design

## Review points

- The styling matches the Figma design

## Screenshots or videos

- figma component
image

- dropdown menu component
image

## Assignee Checklist:

- [x] The base branch is correct (no accidental merges)
- [x] The branch name follows our branch naming rules
- [x] The PR title follows our PR title rules
- [x] My code follows our coding style
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors in the console

## Reviewer Checklist:

- [ ] Code readability and simplicity
- [ ] Follows best practices and coding standards
- [ ] Understandable and maintainable code

### CODE REVIEW

**Comment by HrRn** [comment]
Thank you for your work!
Could you merge the latest develop into this branch?

File: src/components/ui/dropdown-menu.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+'use client';
+
+import * as React from 'react';
```
Comment (HrRn) [inline src/components/ui/dropdown-menu.tsx]: According to the regulations, it is required not to use wildcards. Could you please make the necessary corrections?

Comment (yuki-92c) [reply src/components/ui/dropdown-menu.tsx]: Ah I completely forgot about it. Thank you for the comment. I'll fix that.

**Review by HrRn — APPROVED** [review]
LGTM :D

**Pull request merged**